src/main/java/com/jagrosh/giveawaybot/commands/CreateCommand.java

```diff
@@ -226,14 +227,52 @@ private void waitForPrize(CommandEvent event, TextChannel tchan, int seconds, in
                 return;
             }
 
-            Instant now = Instant.now();
-            if(bot.startGiveaway(tchan, event.getAuthor(), now, seconds, winners, prize))
+            GuildSettingsManager.GuildSettings settings = bot.getDatabase().settings.getSettings(event.getGuild().getIdLong());
+            if (!level.canSetEmoji() && !settings.getEmojiDisplay().equals(Constants.TADA))
             {
-                event.replySuccess("Done! The giveaway for the `"+e.getMessage().getContentRaw()+"` is starting in "+tchan.getAsMention()+"!");
+                event.reply("Hmm... Your custom emoji could not be used.\n\nWould you still like to continue using the `" + Constants.TADA + "` emoji?");
+                waitForEmoji(event, tchan, seconds, winners, prize, lastMessage);
             }
             else
             {
-                event.replyError("Uh oh. Something went wrong and I wasn't able to start the giveaway."+CANCEL);
+                Instant now = Instant.now();
+                if (bot.startGiveaway(tchan, event.getAuthor(), now, seconds, winners, prize)) {
+                    event.replySuccess("Done! The giveaway for the `" + e.getMessage().getContentRaw() + "` is starting in " + tchan.getAsMention() + "!");
+                } else {
+                    event.replyError("Uh oh. Something went wrong and I wasn't able to start the giveaway." + CANCEL);
+                }
+            }
+        });
+    }
+
+    private void waitForEmoji(CommandEvent event, TextChannel tchan, int seconds, int winners, String prize, long lastMessage)
```

Comment:
I'm not a huge fan of this; I think it would be better to just put a notice when they are first setting up the giveaway and let them stop it themselves earlier. For example
> Alright! Let's set up your giveaway! First, what channel do you want the giveaway in?
> You can type `cancel` at any time to cancel creation.
> Note: Your giveaway emoji has been reset since your last giveaway creation

Reply:
so this will just reset the emoji once it is expired using the multi stage setup?